lib/_tokenUtils.ts

```diff
@@ -26,7 +26,12 @@ export const getCookie = (name: string): string => {
   return "";
 };
 
-export function setAnonymousUserToken(): void {
+export function setAnonymousUserToken(transient = false): void {
+  if (transient) {
+    this.setUserToken(`transient-${createUUID()}`);
```

Comment:
I went with the `transient` prefix as a way to distinguish an in-memory user token from a cookie-based one.